### ISSUE TITLE
Implicit values for aria-valuemin and aria-valuemax on the slider role may need more details

### ISSUE THREAD

**Issue by melanierichards** [w3c/aria#988]
On PR #946, @mcking65 pointed out that the implicit values for `aria-valuemin` and `aria-valuemax` on `slider` role do not completely mirror [those given in HTML for the `range` `input` type](https://www.w3.org/TR/html5/sec-forms.html#range-state-typerange), which we reference.

Matt's words:

> This list is missing the conditions where only one is specified and it invalidates the implicit value of the other. For instance, if aria-valuemax is not specified and aria-valuemin is specified as 200. Or, conversely, if aria-valuemin is not specified and aria-valuemax is set to -1.

I'm not finding at the moment where in HTML this is specified (closest I could find is that in some but not all cases [max must not be less than min](https://www.w3.org/TR/html5/sec-forms.html#the-min-and-max-attributes)), but logging this as a separate issue for discussion.

**Comment by jnurthen** [comment]
Bulk closing stale issues - please reopen if necessary
